bin/oref0-setup.sh

```diff
@@ -1052,7 +1059,7 @@ if [[ $REPLY =~ ^[Yy]$ ]]; then
 
     # install Go for Explorer Board/HAT
     if [[ "$ttyport" =~ "spidev" ]] || [[ ${CGM,,} =~ "g4-go" ]]; then
-      if $buildgofromsource; then
+      # if $buildgofromsource; then  # Is preventing go from being downloaded, I think the check is invalid in this position since it's not related to building custom go binaries
```

Comment:
No, I think this was correct.  If you're not building anything from source, you don't want to download and install the (huge) go toolchain: you can just download the precompiled binaries.

Reply:
Ok, yeah that makes sense, we just need to adjust how it is downloading the g4 binaries, since it is currently trying to use `go get` which doesn't work if you don't have go installed